To be merged for 7.1 MTA-4014: Custom rules in analysis Dependency rule is not fired (#74)
migtools/mta-documentation: docs/topics/analysis-dependency-rule-snippet.adoc

// snippet

> .Dependency rule of custom rules in analysis is not fired
> 
> The dependency rule, part of the custom rules, is not fired, and no related migration issue is found. link:https://issues.redhat.com/browse/MTA-4014[(MTA-4014)]

Reviewer: 1. A custom rules file doesn't have to include a dependency rule.
2. Jira bug: https://issues.redhat.com/browse/MTA-3863

```suggestion
.Dependency rule of a custom rules file in analysis is not fired

A custom dependency rule is not fired, and no related migration issue is found. link:https://issues.redhat.com/browse/MTA-3863[(MTA-3863)]
```
